sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -32465,6 +32465,163 @@
                 "multiple": false,
                 "unique": false
               },
+              "conditional": {
```

Comment:
Can you please tell what is the benefit of moving `collection and use of information` part outside of the rich text editor?

![image](https://github.com/bcgov/SIMS/assets/54600590/d50a73b6-9d98-48ec-a124-d93cb8171a96)

I just copied the text into the rich text editor and compared them and I couldn't see a difference.

![image](https://github.com/bcgov/SIMS/assets/54600590/5457a5db-48d0-4d1c-a0e4-8fcb1e036bce)

Reply:
Sorry Dheepak. I'm not sure about the benefit of moving collection and use of information part outside of the rich text editor. It is used outside of the rich text editor in many places, so I just followed the same way.

Reply:
The format is updated to include collection and use of information inside the rich text editor. Please have a look.